### PR TITLE
[feature1] new file added

### PR DESCRIPTION
**Pull request by qaziaqibnawaz**

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -1 +1 @@
-pull request1
```
Comment (qazi-aqib) [inline file1.txt:1]: please make font better

Comment (qaziaqibnawaz) [reply file1.txt:1]: sure

**Review by qazi-aqib — APPROVED** [review]
good job

**Review by qazi-aqib — APPROVED** [review]
good

**Review by qazi-aqib — APPROVED** [review]

**Review by qazi-aqib — APPROVED** [review]
approved

**Pull request merged**